packages/loader/container-loader/src/container.ts

```diff
@@ -294,6 +294,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         loader: Loader,
         loadOptions: IContainerLoadOptions,
         pendingLocalState?: unknown,
+        allowLegacy?: boolean,
```

Comment:
Should it be a property in IContainerLoadOptions instead?
Please add documentation in the form of proper comments